workload/Config.mk

```diff
@@ -1,17 +1,17 @@
-# DOTNET6_VERSION
+# DOTNET7_VERSION
 -include $(TMPDIR)/dotnet-version.config
 $(TMPDIR)/dotnet-version.config: $(TOP)/build/Versions.props
 	@mkdir -p $(TMPDIR)
-	@grep "<MicrosoftDotnetSdkInternalPackageVersion>" build/Versions.props | sed -e 's/<\/*MicrosoftDotnetSdkInternalPackageVersion>//g' -e 's/[ \t]*/DOTNET6_VERSION=/' > $@
-DOTNET6_VERSION_BAND = $(firstword $(subst -, ,$(DOTNET6_VERSION)))
+	@grep "<MicrosoftDotnetSdkInternalPackageVersion>" build/Versions.props | sed -e 's/<\/*MicrosoftDotnetSdkInternalPackageVersion>//g' -e 's/[ \t]*/DOTNET7_VERSION=/' > $@
+DOTNET7_VERSION_BAND = $(firstword $(subst -, ,$(DOTNET7_VERSION)))
 
-# DOTNET6_DESTDIR
+# DOTNET7_DESTDIR
```

Comment:
I think it's better to remove the version number from the variable for later.

Reply:
agreed!